src/toxcore/state_format/old.rs

```diff
@@ -151,6 +160,37 @@ pub struct NospamKeys {
 /// Number of bytes of serialized [`NospamKeys`](./struct.NospamKeys.html).
 pub const NOSPAMKEYSBYTES: usize = NOSPAMBYTES + PUBLICKEYBYTES + SECRETKEYBYTES;
 
+/** E.g.
```

Comment:
In the comments you should never just throw an example at the reader without a short explanation what it does or what it should be an example for :)